src/backend/app/Services/CompanyService.php

```diff
@@ -25,11 +24,6 @@ public function getByDatabaseProxy($databaseProxy): Company {
 
     public function getById($id): Company {
         $result = $this->company->newQuery()->findOrFail($id);
-        if (isset($result->protected_page_password)) {
-            if (str_starts_with($result->protected_page_password, 'eyJ')) {
```

Comment:
Why are we removing this? This should not be affected even if `protected_page_password` is null.

Reply:
Yeah, it was just a leftover I think 🤔 We can continue this when we bring back the feature in a new PR.